typescript/ws-protocol-examples/README.md

```diff
@@ -1,22 +1,36 @@
-# Foxglove Studio WebSocket protocol examples
+# Foxglove WebSocket examples in TypeScript
 
-This package provides example server and client scripts using the [Foxglove Studio WebSocket protocol](https://github.com/foxglove/ws-protocol).
+The Foxglove WebSocket protocol is encoding-agnostic, as long as the desired encoding is supported by both client and server. It currently supports Protobuf and JSON messages.
```

Comment:
I think I intentionally made this intro paragraph a bit different from the others, linking out to the main docs, since it's meant to be an example showcase rather than description of the protocol.